src/Assert.php

```diff
@@ -53,7 +53,7 @@ public static function assertJsonMatchesSchema($schema, $content)
         }, $validator->getErrors());
         $messages[] = '- Response: '.json_encode($content);
 
-        self::assertTrue($validator->isValid(), implode("\n", $messages));
+        \PHPUnit_Framework_Assert::assertTrue($validator->isValid(), implode("\n", $messages));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\PHPUnit_Framework_Assert' in method 'assertJsonMatchesSchema'.](https://www.codacy.com/app/estahn/phpunit-json-assertions/file/2256991334/issues/source?bid=3181605&fileBranchId=3182397#l56)